woocommerce/google-listings-and-ads
# Add blurb to inform users to connect Google Ads child account and not a parent MCC account

### Changes proposed in this Pull Request:



Closes #1346.

This PR adds a "blurb" or message to the UI to encourage users to only connect child Google Ads accounts, not manager account.

The message will only be displayed when there are more than one Google Ads account. Clicking on the "Learn more" link will log a track event: 

```
wcadmin_gla_documentation_link_click {context: 'setup-ads-connect-account', link_id: 'connect-sub-account', href: 'https://support.google.com/google-ads/answer/6139186'}
```

Other technical notes:

- I have changed the `AdsAccountSelectControl` so that it receives `accounts` prop that is passed down from `NonConnected` component and doesn't need to rely on `useExistingGoogleAdsAccounts` hook.
- `AdsAccountSelectControl` is only used in `ConnectAds` component. I have moved it from the shared components folder into the local folder together with `ConnectAds`.

### Screenshots:

When there is only one Google Ads account:

![image](https://user-images.githubusercontent.com/417342/159958059-42a01599-6932-455e-9c1e-e5f1f218642a.png)

When there are more than one Google Ads account: 

![image](https://user-images.githubusercontent.com/417342/160144128-274b2b8e-71f2-427d-b1bd-6eddd3c1f014.png)

Clicking on the "Learn more" link will log a track event: 

![image](https://user-images.githubusercontent.com/417342/159958260-b18378b7-b0ae-4444-b79f-051f9f10ec68.png)

### Detailed test instructions:

Note: to simulate a single Google Ads account or multiple Google Ads accounts, you can modify the following lines:

https://github.com/woocommerce/google-listings-and-ads/blob/e06e6c313d6529b739100cc3fd62534963c4e014/js/src/data/reducer.js#L241-L243

```js
		// For single account:
		case TYPES.RECEIVE_ACCOUNTS_GOOGLE_ADS_EXISTING: {
			return setIn( state, 'mc.accounts.existing_ads', [ 11111 ] );
		}

		// For multiple accounts:
		case TYPES.RECEIVE_ACCOUNTS_GOOGLE_ADS_EXISTING: {
			return setIn( state, 'mc.accounts.existing_ads', [ 11111, 22222 ] );
		}
```

1. Disconnect your Google Ads account.
2. Go to Setup Ads flow. You should see the screenshots above.
3. Click on the "Learn more" link. You should see track event being logged.



### Changelog entry

> Tweak - Add message to advise users to only connect Google Ads child account, not manager account.
